commands/comment.go

```diff
@@ -43,7 +43,13 @@ var (
 )
 
 func init() {
-	commentFlagSet.Var(&commentLocation, "l", "File location be commented upon; requires that the -f flag also be set")
+	commentFlagSet.Var(&commentLocation, "l",
+		`File location be commented upon; requires that the -f flag also be set.
```

Comment:
It looks like there was a grammar error in the original doc that has been copied over here.

Instead of "location be commented upon" it should either say "location commented upon" or "location to be commented upon".